bin/MadGraph5_aMCatNLO/cards/production/13TeV/SinglyChargedHiggsGMmodel_HWZ_M1000/SinglyChargedHiggsGMmodel_HWZ_M1000_run_card.dat

```diff
@@ -277,5 +277,5 @@
 30 50 = sys_matchscale # variation of merging scale
 # PDF sets and number of members (0 or none for all members).
 # CT10nlo.LHgrid = sys_pdf # matching scales
-NNPDF30_lo_as_0130.LHgrid = sys_pdf
+CT10nlo.LHgrid = sys_pdf
```

Comment:
yes, not planning to use different pdf sets, just weight variations of the default one. This does not interfere right?